app/components/Game/Game.js

```diff
@@ -16,34 +19,71 @@ export default function Game({
     endButtonPressed,
     goButtonPressed,
   },
-  handleClicks,
+  handleClicks: {
+    handleClickOnBoard,
+    toggleStartButtonPressed,
+    toggleEndButtonPressed,
+    findAndDrawPath,
+  },
 }) {
-  const gameHeaderState = {
+  const instructionsState = {
     pathStartingPoint,
     pathEndingPoint,
     startButtonPressed,
     endButtonPressed,
   };
-  const gameScreenState = {
+  const instructionsProps = { instructionsState, };
+  const instructions = (
+    <div className="instructionsWrapper" key="2">
+      <GameInstructions { ...instructionsProps } />
+    </div>
+  );
+  const settingsText = '<-- Settings';
+  const linkProps = {
+    className: 'settingsLink',
+    to: { pathname: '/settings', },
+    children: settingsText,
```

Comment:
I'd inline it

Reply:
you mean `children: '<---settings'`?

Reply:
Yes

Reply:
done

(also moved between `<Link></Link>` )